scripts/tcl_commands/routing.tcl

```diff
@@ -459,23 +431,13 @@ proc run_resizer_timing_routing {args} {
         set log [index_file $::env(routing_logs)/resizer.log]
         puts_info "Running Global Routing Resizer Timing Optimizations (log: [relpath . $log])..."
 
-        set ::env(SAVE_DEF) [index_file $::env(routing_tmpfiles)/resizer_timing.def]
-        set ::env(SAVE_SDC) [index_file $::env(routing_tmpfiles)/resizer_timing.sdc]
-
-        run_openroad_script $::env(SCRIPTS_DIR)/openroad/resizer_routing_timing.tcl -indexed_log $log
-        set_def $::env(SAVE_DEF)
-        set ::env(CURRENT_SDC) $::env(SAVE_SDC)
+        run_openroad_script $::env(SCRIPTS_DIR)/openroad/resizer_routing_timing.tcl\
+            -indexed_log [index_file $::env(routing_logs)/resizer.log]\
+            -save "dir=$::env(routing_tmpfiles),def,sdc,odb,netlist,powered_netlist"
 
         TIMER::timer_stop
         exec echo "[TIMER::get_runtime]" | python3 $::env(SCRIPTS_DIR)/write_runtime.py "resizer timing optimizations - openroad"
 
-        write_verilog $::env(routing_results)/$::env(DESIGN_NAME).resized.v -log $::env(routing_logs)/write_verilog.log
```

Comment:
same comment as the previous removal

Reply:
You'll find the `-save` argument a couple lines above now has : `netlist,powered_netlist`.